README.md

```diff
@@ -49,6 +49,12 @@ However, its primary use case is to notify players about the status of the tourn
 There are plans for many more services in the future.
 To learn more, see our [future plans](##Current-and-Future-State).
 
+## Contributing
+To contribute to the project, create branches off the repository itself (don't fork into your own namespace) and open PR's against the main branch. All PR's are subject to review. To run the project in development, execute the following commands:
+
+cargo install cargo-shuttle
```

Comment:
Let's format these commands as a code block using triple backticks, i.e. `

Reply:
Done! Thanks for the advice. :)

Reply:
As to why the .history directory was added, was this not a part of the project already? I didn't add the directory, I'm not sure what I would have done to do so. Should it be included in the .gitignore?

Reply:
> As to why the .history directory was added, was this not a part of the project already?

Correct, they were not part of the repo before. Perhaps, `.history` should be added. For now, just delete the directory and if this happens again, we can dig deeper.

Reply:
Okay, .history is now in the gitignore. The problem should be resolved.